docs/_providers/powerdns.md

```diff
@@ -54,4 +54,4 @@ See the [PowerDNS documentation](https://doc.powerdns.com/authoritative/http-api
 
 ## Caveats
```

Comment:
This can be removed. On-demand activation/deactivation has been implemented with AUTODNSSEC.